guide/inheritance-wallet/succession.md

```diff
@@ -0,0 +1,229 @@
+
+4. **Recoverability**: The timelocked recovery path provides a way for owners to regain access to their funds if they lose some of their keys, without compromising security under normal circumstances
+
+5. **Gradual transition**: The setup allows for a gradual transition of control from the original owners to the heirs, which can be beneficial for estate planning and teaching financial responsibility
+
+6. **Legal integration**: The design allows for the involvement of legal professionals (like Edward in the example), providing a bridge between bitcoin inheritance and traditional legal processes
+
+### Cons
+
+1. **Complexity**: The multi-key and timelock setup is more complex than a simple single-signature wallet, which may be challenging for less technically inclined users to understand and manage
+
+2. **Maintenance**: Regular key checks and potential updates to the wallet configuration require ongoing attention and effort from the wallet owners
+
+3. **Backup challenges**: With multiple keys and a wallet descriptor to back up, there are more components to secure and potentially lose compared to a single-key wallet
+
+4. **Coordination required**: In scenarios where multiple signatures are needed, coordinating between different signers can be logistically challenging, especially in time-sensitive situations
+
+5. **Potential for errors**: The complexity of the setup increases the risk of user errors, such as misconfiguring timelocks or mismanaging keys, which could lead to loss of funds
+
+6. **Limited support**: Not all wallet software supports complex multi-key and timelock configurations, potentially limiting users' options for managing their funds
+
+7. **Inheritance key holder privacy considerations**: While inheritance key holders can mitigate privacy risks by generating new XPUBs or using dedicated devices for the inheritance wallet, this adds another layer of complexity and potential cost
+
+8. **Time delay for full inheritance access**: While heirs with access to a primary key could potentially use the recovery path for quicker access, full control via inheritance keys is still subject to the timelock delay
+
+9. **Ongoing transaction costs**: Timelocks need to be refreshed at regular intervals, incurring transaction fees. This ongoing cost needs to be factored into the long-term management of the wallet
+
+In conclusion, while multi-key and timelocks offer powerful tools for creating secure and flexible inheritance solutions, they also require careful planning, ongoing management, and a good understanding of the technology. Families considering such a setup should weigh these pros and cons against their specific needs, technical capabilities, and risk tolerance. The added security and flexibility come with increased complexity and maintenance requirements, including regular updates and associated transaction costs.
```

Comment:
Is the amount of complexity more or less than traditional inheritance setups? Here's what Claude had to say:

> The bitcoin setup is technically more complex but operationally simpler in many ways. It eliminates many intermediaries and provides more direct control, but requires:
> 
> - Technical competence from all parties
> - Careful key management
> - Regular maintenance (checking keys, updating configurations)
> - Family coordination and education
> 
> I think we should update the document to include this comparison explicitly - it would help readers understand the tradeoffs they're making. It's not necessarily more or less complex overall, but the complexity shifts from legal/bureaucratic to technical/operational.